ios/Plugin/CleverPushCapacitorPlugin.m

```diff
@@ -73,24 +73,24 @@ - (void)getSubscriptionId:(CAPPluginCall *)call {
 }
 
 - (void)trackPageView:(CAPPluginCall *)call {
-    NSString *value = [call getString:@"trackPage"];
+    NSString *value = [call getString:@"url"];
     [CleverPush trackPageView:value];
 }
 
 - (void)addSubscriptionTag:(CAPPluginCall *)call {
-    NSString *value = [call getString:@"subscriptionTag"];
+    NSString *value = [call getString:@"tagId"];
     [CleverPush addSubscriptionTag:value];
 }
 
 - (void)removeSubscriptionTag:(CAPPluginCall *)call {
-    NSString *value = [call getString:@"subscriptionTag"];
+    NSString *value = [call getString:@"tagId"];
```

Comment:
Not sure if this will work now, because we migrated the object argument to only-string. Please test